### ISSUE TITLE
[Notion] Cannot open page after creating it

### ISSUE THREAD

**Issue by cpa** [raycast/extensions#9152]
### Extension

https://www.raycast.com/notion/notion

#### Description
Cannot open page after creating it (Cmd-O throws this error)

#### Error:
```
Error: Minified React error #321

### Steps To Reproduce

1. Install and authorize notion plugin to access at least one db
2. Create a notion page using action "Create database page"
3. After creation, click on "Open Page" or type Cmd-O

### Current Behaviour

Throws the error above

### Expected Behaviour

Open notion app (or browser if app is not installed) in the newly created page.

**Comment by erics118** [comment]
I don't think there's an API for one extension to uninstall another extension, so I believe this isn't officially possible. There may be some hacky way to do it though

**Comment by AlexIsMaking** [comment]
I've just checked and I was able to reproduce this issue.

**Comment by erics118** [comment]
This issue should be fixed in #9182

**Comment by cpa** [comment]
That's great, thanks!
